docs/manage-persistent-storage.md

```diff
@@ -0,0 +1,391 @@
+- will detach storage data/4
+
+Continue [y/N]? y
+```
+
+As you can see, the `logs` storage will be removed while the `data` storage will only be detached.
+
+You can check this after removing the units by running the `juju status --storage` command:
+```shell
+Model  Controller      Cloud/Region         Version  SLA          Timestamp
+etcd   dev-controller  localhost/localhost  3.6.0    unsupported  13:02:34Z
+
+App           Version  Status   Scale  Charm         Channel  Rev  Exposed  Message
+charmed-etcd           unknown      0  charmed-etcd             0  no       
+
+Storage Unit  Storage ID  Type        Pool          Mountpoint  Size     Status    Message
+              data/0      filesystem  etcd-storage              8.0 GiB  detached  
+              data/2      filesystem  etcd-storage              8.0 GiB  detached  
+              data/4      filesystem  etcd-storage              8.0 GiB  detached  
+```
+
+When you want to scale up your etcd database again, attach the desired `data` volume to the new unit of charmed etcd:
+```shell
+juju add-unit charmed-etcd --attach-storage=data/0
+```
+
+If you did not remove all previous units of charmed etcd at the same time, make sure to attach the volume with most 
+recent data (meaning: of the unit you removed last).
+
+You can track the progress by running `juju status --storage --watch=3s`:
+```shell
+Model  Controller      Cloud/Region         Version  SLA          Timestamp
+etcd   dev-controller  localhost/localhost  3.6.0    unsupported  13:06:22Z
+
+App           Version  Status  Scale  Charm         Channel  Rev  Exposed  Message
+charmed-etcd           active      1  charmed-etcd             0  no       
+
+Unit             Workload  Agent  Machine  Public address  Ports  Message
+charmed-etcd/3*  active    idle   3        10.105.253.200         
+
+Machine  State    Address         Inst id        Base          AZ  Message
+3        started  10.105.253.200  juju-84b57b-3  ubuntu@24.04      Running
+
+Storage Unit    Storage ID  Type        Pool          Mountpoint                                  Size     Status    Message
+                data/2      filesystem  etcd-storage                                              8.0 GiB  detached  
+                data/4      filesystem  etcd-storage                                              8.0 GiB  detached  
+charmed-etcd/3  data/0      filesystem  etcd-storage  /var/snap/charmed-etcd/common/var/lib/etcd  8.0 GiB  attached  
+charmed-etcd/3  logs/6      filesystem  rootfs        /var/snap/charmed-etcd/common/var/log/etcd  76 GiB   attached  
+```
+
+As you can see, the new unit `charmed-etcd/3` has been deployed with the existing `data/0` volume and the new volume
+`logs/6` attached to it.
+
+In order to scale up to three units again, add execute two more `juju add-unit` commands with the other storage volumes:
+```shell
+juju add-unit charmed-etcd --attach-storage=data/2
+juju add-unit charmed-etcd --attach-storage=data/4
+```
+
```

Comment:
please add a note here that the data in these 2 volumes will not be taken into account given that they will be overriden by the content of the first volume.